vortex/vortex-tools.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/bash
+printf "%s\n" "INFO: Running Pikdum's Tools for Vortex";
+printf "%s\n" "INFO: Checking for updates";
+PIKDUMSVERSIONNEW=$(curl https://raw.githubusercontent.com/SirStig/Steam-deck-tests/main/CHANGELOG.md);
```

Comment:
**suggestion:** I'd revert all the version checking and updating changes, just to keep things simpler.

If people want to update, they can run the update shortcut. Otherwise, they can just stay on whatever version works.

Reply:
My thought's for checking for updates was it's simple, there's really no reason for it not to, and the main reason is because most people don't really pay attention to things. If theres big changes or updates to pikdum's no ones going to see as no one comes here afterwards unless they have an issue, and no one really things about it. It's more to help people.

Like if theres new support or features that could help someone/make something simpler or easier or fix issues they never cared to complain about. It'll just always be kept up to date

Reply:
I'd rather keep updating as an explicit action, not as a side effect.